localstack/utils/aws/aws_responses.py

```diff
@@ -59,13 +60,37 @@ def requests_error_response_xml(
     return response
 
 
+def to_xml(data: dict, memberize: bool = True) -> ET.Element:
+    """Generate XML element hierarchy out of dict. Wraps list items in <member> tags by default"""
+    if type(data) != dict or len(data.keys()) != 1:
+        raise
+
+    def _to_xml(parent_el: ET.Element, data_rest) -> None:
+        if type(data_rest) == list:
+            if len(data_rest) == 0:
```

Comment:
Good catch. I had some code here previously that was handling different ways to deal with empty elements but removed it for now until I get to a use case where I can test it properly. 

E.g. some responses render "empty elements" as missing, some truncate them to `<a />`, etc. This might be something that is specific to the different APIs though and sometimes trips up the boto (de-)serializer otherwise.